docs/openid.md

```diff
@@ -238,13 +238,36 @@ Synapse config:
 
 ```yaml
 oidc_config:
-   enabled: true
-   issuer: "https://id.twitch.tv/oauth2/"
-   client_id: "your-client-id" # TO BE FILLED
-   client_secret: "your-client-secret" # TO BE FILLED
-   client_auth_method: "client_secret_post"
-   user_mapping_provider:
-     config:
-       localpart_template: '{{ user.preferred_username }}'
-       display_name_template: '{{ user.name }}'
+  enabled: true
+  issuer: "https://id.twitch.tv/oauth2/"
+  client_id: "your-client-id" # TO BE FILLED
+  client_secret: "your-client-secret" # TO BE FILLED
+  client_auth_method: "client_secret_post"
+  user_mapping_provider:
+    config:
+      localpart_template: "{{ user.preferred_username }}"
+      display_name_template: "{{ user.name }}"
```

Comment:
Not sure if we want to change these from single to double quotes. Single quotes will prevent many characters from being escaped in yaml.

Reply:
I changed it to be consistent with the above examples, should have pointed that out! I can revert it if you want. I don't think any of the characters in these strings are escaped though so it should be OK?

Reply:
Ah, yes I see. Yes, that should be fine, and only double-quotes are needed for `{` and `}`.

Seems good then!